builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -285,8 +285,18 @@ def unDeployService(stage) {
 				env_key = "PROD"
 			}
 			def envBucketKey = "${env_key}${configLoader.JAZZ.S3_BUCKET_NAME_SUFFIX}"
-			sh "serverless remove --stage ${stage} --verbose  --bucket ${configLoader.AWS.S3[envBucketKey]}"
-
+			//empty bucket before deleting it. //			
+			if(service_config['event_source_s3'] && (service_config['event_source_s3']!= null)){
```

Comment:
Is this working for dev environments ?? I said to check the bucket name for multi environments since we are creating s3 bucket per environments.  Also we need to check before deleting the content of the bucket , as it is as part of the service or not